Project.toml

```
@@ -10,14 +10,16 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
 Arrow = "2"
+Compat = "3.34"
```

can we add a defensive compat 4 compat here too?
```suggestion
Compat = "3.34, 4"
```